tf_big/python/ops/big_ops_test.py

```diff
@@ -46,21 +48,57 @@ def test_add(self):
 
   def test_2d_matrix_add(self):
     with tf.Session() as sess:
-      a = np.array([[b"5", b"5"], [b"5", b"5"]])
-      b = np.array([[b"6", b"6"], [b"6", b"6"]])
+      a = np.array([[5, 5], [5, 5]]).astype(np.int32)
+      b = np.array([[6, 6], [6, 6]]).astype(np.int32)
 
-      expected = a.astype(np.int32) + b.astype(np.int32)
+      expected = a + b
 
       a_var = big_import(a)
       b_var = big_import(b)
 
       c_var = big_add(a_var, b_var)
 
-      c_str = big_export(c_var, tf.string)
+      c_str = big_export(c_var, tf.int32)
+
+      output = sess.run(c_str)
+
+      np.testing.assert_equal(output, expected)
+
+  def test_matmul(self):
+    with tf.Session() as sess:
+      a = np.array([[5, 5], [5, 5]]).astype(np.int32)
+      b = np.array([[6, 6], [6, 6]]).astype(np.int32)
+
+      expected = a.dot(b)
```

Comment:
`dot` is from Eigen? TF seems to exclusively use `matmul`

Reply:
this is just numpy, maybe it should say `matmul`?

Reply:
ah right, nvm then :)